### PR TITLE
bug fixed and refresh - part2

### PR DESCRIPTION
**Pull request by 0utsider**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,48 +1,37 @@
...
-  lineinfile: dest=/etc/ferm.conf line="@include 'ferm/ferm.d/';" backup=yes insertbefore=BOF create=yes
-  when: ansible_os_family == "RedHat"
-
-- name: Ferm | Create the default ferm conf files
-  template: src=ferm.conf.j2 dest=/etc/ferm/ferm.d/{{item.key}}.conf mode=0655 owner=root group=root backup=yes
-  with_dict: ferm_rules_defaults
-  notify:
-    - reload ferm
+  lineinfile: dest=/etc/ferm.conf line="@include 'ferm.d/';" backup=yes insertbefore=BOF create=yes
+  when: ansible_os_family|lower != "redhat"
```
Comment (ant31) [inline tasks/main.yml]: should be == redhat I think?

File: defaults/main.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ferm_rules_defaults:
```
Comment (ant31) [inline defaults/main.yml:1]: we expect user to override ferm_rules variables, by doing so if 'mergeDict' option isn't set to true, user will lose the defaults setup.

This is why there is 2 differents values, it came from a previous PR.

Comment (ant31) [reply defaults/main.yml:1]: commit: 
https://github.com/ant31-ansible/role-ferm-firewall/commit/8afc3d485ca640830aa3d79f7cfc159de1d045ea


 - ferm_rules has a default and is overwritten if defined elsewhere
 - ferm_rules_extra can be defined and if so those rules will be added
   in addition to the ones defined in ferm_rules

Comment (0utsider) [reply defaults/main.yml:1]: You right - but better way to make that is use a compare().
Look my new commit

File: tasks/main.yml (hunk 1 above)
Comment (0utsider) [reply tasks/main.yml]: yes - my misspell - I fixed in next commit.

**Comment by ant31** [comment]
LG, Thanks !

**Pull request merged**